src/domain/community/community/HomePageContributors/queries.graphql

```diff
@@ -1,28 +1,36 @@
-query contributingUsers {
-  usersPaginated(first: 25) {
-    users {
+query contributingUsers($limit: Float, $shuffle: Boolean, $filterCredentials: [AuthorizationCredential!]) {
+  users(limit: $limit, shuffle: $shuffle, filter: { credentials: $filterCredentials}) {
+    id
+    nameID
+    displayName
+    profile {
       id
-      displayName
-      nameID
-      profile {
+      location {
+        city
+        country
+      }
+      avatar {
         id
-        location {
-          city
-          country
-        }
-        avatar {
-          id
-          uri
-        }
-        tagsets {
-          id
-          tags
-        }
+        uri
+      }
+      tagsets {
+        id
+        tags
       }
-    }
-    pageInfo {
-      endCursor
-      hasNextPage
     }
   }
 }
+
```

Comment:
this is copied from src/domain/community/contributor/organization/graphql/queries/organizationsList.graphql not moved because it's used in other places too